dependencies.yaml

```diff
@@ -169,6 +169,9 @@ main_requires:
   perl(Try::Tiny):
   perl(XML::LibXML):
   perl(XML::SemanticDiff):
+  perl(YAML::PP):
```

Comment:
this duplicates that previously we already required it from https://github.com/os-autoinst/os-autoinst/pull/2004/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6R109 . I suggest to remove https://github.com/os-autoinst/os-autoinst/pull/2004/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6R109 and keep this more low-level dependency

Reply:
The links don't work for me, but I guess the suggestion is to remove YAML::PP from test_requires, now that it is added to main_requires.

Reply:
maybe now you need to manually remove the double `perl(YAML::PP)` from Dockerfile and cpanfile and spec file